.github/workflows/scrape.yml

```diff
@@ -40,7 +54,7 @@ jobs:
           GH_PAT: ${{ secrets.GH_PAT }}
         run: |
           cd scraper
-          mv -f ./timetable.json ../public/
+          mv -f ./timetable.json ../public/timetable_${YEAR}_${SESSION#* }.json
```

Comment:
Had to do a google search to grok this, a comment might help. To my understanding this is getting the part of `SESSION` after the first space, eg `S1` from `1 S1`, correct?

Reply:
Oh sorry, yes I was supposed to put a comment here